clients/cobol-lsp-vscode-extension/src/services/LanguageClientService.ts

```diff
@@ -43,11 +44,23 @@ export class LanguageClientService {
     }
 
     public start(): vscode.Disposable {
-        const languageClient = new LanguageClient(LANGUAGE_ID,
-            "LSP extension for " + LANGUAGE_ID + " language",
-            this.createServerOptions(this.jarPath),
-            this.createClientOptions());
-        return languageClient.start();
+        return this.getLanguageClient().start();
```

Comment:
Reverting back custom thread pool changes for this class.

Reason:
Manage thread for the process we initiate and let LSP4J manage its thread. This helps in shutdown process where we shutdown the custom thread pool to shutdown all current processes.